docs/intro.md

```diff
@@ -2,62 +2,48 @@
+This project is based on the article: Saputra, F.A.; Salman, M.; Hasim, J.A.N.; Nadhori, I.U.; Ramli, K. The Next-Generation NIDS Platform: Cloud-Based Snort NIDS Using Containers and Big Data. Big Data Cogn. Comput. 2022, 6, 19.
+[Read the article here](https://www.mdpi.com/2504-2289/6/1/19)
 
-## Project Overview
+## Key Features
 
-![MataElang-v2-Architecture](/img/MataElangv2Architecture.png)
+- 🚀 **Effortless Setup**: Mata Elang provides a seamless deployment experience using Docker Compose, making it easy to install and manage across different environments. Whether you're deploying on a single machine or a cloud-based infrastructure, the setup process is streamlined to reduce complexity and ensure a quick start.
+- 🔍 **Advanced Intrusion Detection with Snort**: At its core, Mata Elang integrates Snort, a leading Network Intrusion Detection System (NIDS), to monitor and analyze network traffic for potential threats. It leverages real-time packet inspection and rule-based detection to identify malicious activities and security breaches.
+- 📊 **Big Data Analytics for Threat Intelligence**: Unlike traditional NIDS solutions, Mata Elang is built on a Big Data platform, allowing it to process and store massive amounts of network traffic data efficiently. This ensures better detection accuracy, supports long-term trend analysis, and enables real-time correlation of security events.
+- 🏗 **Scalable & Distributed Architecture**: Mata Elang is designed for scalability, enabling security teams to deploy sensors across multiple network nodes. Its distributed architecture ensures that even high-traffic networks can be monitored efficiently without performance bottlenecks.
+- 🛡 **Real-Time Threat Detection & Response**: By leveraging machine learning and behavioral analysis, Mata Elang can detect anomalies in network traffic and provide real-time alerts. The system is capable of adapting to new and evolving cyber threats, offering a proactive approach to network security.
 
-Mata Elang is the evolution of the Mata Garuda Internet Monitoring Project for Indonesia. This project was initialized as a private repository in 2018 by LabJarkomC307 - Politeknik Elektronika Negeri Surabaya. Currently, Mata Elang is a collaboration research between PENS, Universitas Indonesia, and BRIN (Badan Riset dan Inovasi Nasional). Mata Elang is supported by JICA.
+## Project Architecture
 
-This project is based on the article: [The Next-Generation NIDS Platform: Cloud-Based Snort NIDS Using Containers and Big Data](https://www.mdpi.com/2504-2289/6/1/19)
+![MataElang-v2-Architecture](/img/MataElangv2Architecture.drawio__2_.png)
```

Comment:
The image already exists in `static/uploads/895f8b2042c298e66625e99e20c8a409/MataElangv2Architecture.drawio__2_.png`